### PR TITLE
ui design

### PR DESCRIPTION
**Pull request by mim24sd**
make html & css of weather forecasting

### CODE REVIEW

File: assests/css/index.css [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+    url(../images/background.jpg);
+  background-size: cover;
+  min-height: 100vh;
+  color: var(--lightColor);
+  align-items: center;
+  font-family: var(--primaryFont);
+}
+
+.header {
+  width: 100%;
```
Comment (takrami) [inline assests/css/index.css]: why do we need this?

File: index.html [hunk 2]
```diff
@@ -1,12 +1,78 @@
...
   </head>
   <body>
-    <p>hi</p>
+    <header class="header">
+      <i class="fa fa-bolt fa-magnifying-glass"></i>
+      <input class="header-search" type="text" placeholder="Search.." >
+    </header>
+    <main class="main">
+      <section class="city-section">
+        <h2 class="city-name">Tehran</h2>
```
Comment (takrami) [inline index.html]: please do not skip heading levels.

**Review by takrami — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (mim24sd) [reply index.html]: Sure Thanks.

File: assests/css/index.css (hunk 1 above)
Comment (mim24sd) [reply assests/css/index.css]: WOW I deleted and it worrrrrkeeeeeedddddd
Thanks

**Review by takrami — APPROVED** [review]
well done! :)

**Pull request merged**